client/src/App.js

```diff
@@ -1,11 +1,158 @@
 import "./App.css";
+import Button from "react-bootstrap/Button";
+import Form from "react-bootstrap/Form";
+import InputGroup from "react-bootstrap/InputGroup";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import Video from "./Video.jsx";
+import React, { useState, useEffect } from "react";
 
 function App() {
+  const [showModal, setShowModal] = useState(false);
+  const [videos, setVideos] = useState([]);
+  const [newVideoTitle, setNewVideoTitle] = useState("");
+  const [newVideoUrl, setNewVideoUrl] = useState("");
+
+  useEffect(() => {
+    fetchVideos();
+  }, []);
+
+  function fetchVideos() {
+    fetch("https://video-server-wsp9.onrender.com/videos")
+      .then((res) => res.json())
+      .then((data) => setVideos(data))
+      .catch((error) => console.log(error));
+  }
+
+  function handleShowModal() {
+    setShowModal(!showModal);
+  }
+
+  function handleDelete(id) {
+    fetch(`https://video-server-wsp9.onrender.com/videos/${id}`, {
+      method: "DELETE",
+      headers: {
+        "Content-Type": "application/json",
+      },
+    })
+      .then((res) => {
+        if (res.ok) {
+          return res.json();
+        } else {
+          return {};
+        }
+      })
+      .then((deleteVideo) => {
+        const updatedVideos = videos.filter(
+          (video) => video.id !== deleteVideo.id
+        );
+        setVideos(updatedVideos);
+      })
+      .catch((error) => console.log(error));
+  }
+
+  function handleAddVideo() {
+    if (newVideoTitle.trim() === "") {
+      alert("Please enter a valid title.");
+      return;
+    }
+
+    if (!newVideoUrl.includes("youtube.com")) {
+      alert("Please enter a valid YouTube URL.");
+      return;
+    }
+
+    // const newVideoId = Date.now().toString(); // Generate unique ID using current timestamp
```

Comment:
try to clean up code you no longer need! 😄

Reply:
Ok , thank you Sam , I'll delete it